[35mDEBUG   [0m tests.conftest:conftest.py:51 Running fixture setup: test_id
[35mDEBUG   [0m tests.conftest:conftest.py:57 Running test: test_combination_of_different_nodes with id: 2024-10-29_04-08-48__1fdfe0dc-43de-4e01-8b69-3c893b9a442e
[35mDEBUG   [0m src.steps.common:common.py:18 Running fixture setup: common_setup
[35mDEBUG   [0m src.steps.light_push:light_push.py:27 Running fixture setup: light_push_setup
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/receiving_node1_2024-10-29_04-08-48__1fdfe0dc-43de-4e01-8b69-3c893b9a442e__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.189.9
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['12343', '12344', '12345', '12346', '12347']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:434 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:163 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:165 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 12343:12343 -p 12344:12344 -p 12345:12345 -p 12346:12346 -p 12347:12347 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=12345 --rest-port=12343 --tcp-port=12344 --discv5-udp-port=12346 --rest-address=0.0.0.0 --nat=extip:172.18.189.9 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=f3cab24decd6599a225bf042caba889b30d1e129fabb5e7e9da0c7202f286faa --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=12347 --metrics-logging=true --lightpush=true --relay=true --filter=true
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.189.9 waku 3b76270051ef7963ffc6047803a806fdaee87f6cf86b51cf7126211ad380931f
[1m[31mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container a0329ec7908c. Exiting log stream.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID 3b76270051ef. Setting up logs at ./log/docker/receiving_node1_2024-10-29_04-08-48__1fdfe0dc-43de-4e01-8b69-3c893b9a442e__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:177 Started container from image wakuorg/nwaku:latest. REST: 12343
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:12343/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:277 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:12343/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.189.9/tcp/12344/p2p/16Uiu2HAmEMuHPouTm1aMEyDqseFEiEK9WMHoisqCqeYX82MwTe3g","/ip4/172.18.189.9/tcp/12345/ws/p2p/16Uiu2HAmEMuHPouTm1aMEyDqseFEiEK9WMHoisqCqeYX82MwTe3g"],"enrUri":"enr:-L24QBv2HcY4CFFxvgoZDWxwdW7-06Fg7-mb_HCn9rj-wPvST2UtJoSBaKKD6uStOn1_E9vsGWcb8EHT32PR1BbtL-oCgmlkgnY0gmlwhKwSvQmKbXVsdGlhZGRyc5YACASsEr0JBjA4AAoErBK9CQYwOd0DgnJzhQADAQAAiXNlY3AyNTZrMaEDGVbFSCUTFtsSKYOcWdvzIKRNq0t9hlEP75q9u_ArWH-DdGNwgjA4g3VkcIIwOoV3YWt1Mg0"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 REST service is ready !!
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/receiving_node2_2024-10-29_04-08-48__1fdfe0dc-43de-4e01-8b69-3c893b9a442e__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.180.221
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['28605', '28606', '28607', '28608', '28609']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:434 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:163 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:165 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 28605:28605 -p 28606:28606 -p 28607:28607 -p 28608:28608 -p 28609:28609 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=28607 --rest-port=28605 --tcp-port=28606 --discv5-udp-port=28608 --rest-address=0.0.0.0 --nat=extip:172.18.180.221 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=c604daf2c0a66cfa66bc140dec7f2cdf4cc39ab9bb45d97df49b81e224d5cadd --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=28609 --metrics-logging=true --lightpush=false --relay=false --filternode=/ip4/172.18.189.9/tcp/12344/p2p/16Uiu2HAmEMuHPouTm1aMEyDqseFEiEK9WMHoisqCqeYX82MwTe3g
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.180.221 waku f0628d34ade2ece805c2cf43527a7b196bab75a99a2cd73a8c863b09101b1418
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID f0628d34ade2. Setting up logs at ./log/docker/receiving_node2_2024-10-29_04-08-48__1fdfe0dc-43de-4e01-8b69-3c893b9a442e__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:177 Started container from image wakuorg/nwaku:latest. REST: 28605
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:28605/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:277 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:28605/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.180.221/tcp/28606/p2p/16Uiu2HAm2gpPkExoZnGzAWk3kh9VwZjDVwrwH4tqSizk24xZK3Eu","/ip4/172.18.180.221/tcp/28607/ws/p2p/16Uiu2HAm2gpPkExoZnGzAWk3kh9VwZjDVwrwH4tqSizk24xZK3Eu"],"enrUri":"enr:-L24QAR3ncFB3TAy-iZ-gfH59doosmlu46qdiO4kGsZHO3lxFOG9ZiVN3UVjqF0KpmHmSOe3NIOeuDgNeB3OcDU0m1gCgmlkgnY0gmlwhKwStN2KbXVsdGlhZGRyc5YACASsErTdBm--AAoErBK03QZvv90DgnJzhQADAQAAiXNlY3AyNTZrMaECa-MIgERAFUBtkDmzN63MWDZr5Yti1LmbAxU4gvUn-d6DdGNwgm--g3VkcIJvwIV3YWt1MgA"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:28605/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.189.9/tcp/12344/p2p/16Uiu2HAmEMuHPouTm1aMEyDqseFEiEK9WMHoisqCqeYX82MwTe3g"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/lightpush_node1_2024-10-29_04-08-48__1fdfe0dc-43de-4e01-8b69-3c893b9a442e__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.58.143
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['13189', '13190', '13191', '13192', '13193']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:434 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:163 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:165 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 13189:13189 -p 13190:13190 -p 13191:13191 -p 13192:13192 -p 13193:13193 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=13191 --rest-port=13189 --tcp-port=13190 --discv5-udp-port=13192 --rest-address=0.0.0.0 --nat=extip:172.18.58.143 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=ffd1fc1d882d507b6d51887e710d10a2fc184b4afbb4db6c0be3b87fbabdfda6 --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=13193 --metrics-logging=true --discv5-bootstrap-node=enr:-L24QBv2HcY4CFFxvgoZDWxwdW7-06Fg7-mb_HCn9rj-wPvST2UtJoSBaKKD6uStOn1_E9vsGWcb8EHT32PR1BbtL-oCgmlkgnY0gmlwhKwSvQmKbXVsdGlhZGRyc5YACASsEr0JBjA4AAoErBK9CQYwOd0DgnJzhQADAQAAiXNlY3AyNTZrMaEDGVbFSCUTFtsSKYOcWdvzIKRNq0t9hlEP75q9u_ArWH-DdGNwgjA4g3VkcIIwOoV3YWt1Mg0 --lightpushnode=/ip4/172.18.189.9/tcp/12344/p2p/16Uiu2HAmEMuHPouTm1aMEyDqseFEiEK9WMHoisqCqeYX82MwTe3g --lightpush=true --relay=false
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.58.143 waku b88318455ce1b6a50b1a00500cb1b5de341572a1937aeaf2576a901c61e4805b
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID b88318455ce1. Setting up logs at ./log/docker/lightpush_node1_2024-10-29_04-08-48__1fdfe0dc-43de-4e01-8b69-3c893b9a442e__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:177 Started container from image wakuorg/nwaku:latest. REST: 13189
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:13189/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:277 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:13189/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.58.143/tcp/13190/p2p/16Uiu2HAmTHFNJnm9paTGs9zn89QbvA4iLLLKVECHyvtjAEStvFgL","/ip4/172.18.58.143/tcp/13191/ws/p2p/16Uiu2HAmTHFNJnm9paTGs9zn89QbvA4iLLLKVECHyvtjAEStvFgL"],"enrUri":"enr:-L24QMJR7-6SEbt83MD30FSyGVEbcVRosXjE_gQOmBMPkM-ZQuCCY6EnA0mr3Vk19QKsc70IeN_F-LAo2_McB2LbyMgCgmlkgnY0gmlwhKwSOo-KbXVsdGlhZGRyc5YACASsEjqPBjOGAAoErBI6jwYzh90DgnJzhQADAQAAiXNlY3AyNTZrMaED2U2BqT0FeWFCu7qMaCXHlwPqv8LpRT5NgLaA_rU1vbmDdGNwgjOGg3VkcIIziIV3YWt1Mgg"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:13189/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.189.9/tcp/12344/p2p/16Uiu2HAmEMuHPouTm1aMEyDqseFEiEK9WMHoisqCqeYX82MwTe3g"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:13189/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.180.221/tcp/28606/p2p/16Uiu2HAm2gpPkExoZnGzAWk3kh9VwZjDVwrwH4tqSizk24xZK3Eu"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:19 Docker client initialized with image wakuorg/nwaku:latest
[35mDEBUG   [0m src.node.waku_node:waku_node.py:85 WakuNode instance initialized with log path ./log/docker/lightpush_node2_2024-10-29_04-08-48__1fdfe0dc-43de-4e01-8b69-3c893b9a442e__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:89 Starting Node...
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:22 Attempting to create or retrieve network waku
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:25 Network waku already exists
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:108 Generated random external IP 172.18.203.244
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:101 Generated ports ['46938', '46939', '46940', '46941', '46942']
[35mDEBUG   [0m src.node.waku_node:waku_node.py:434 RLN credentials were not set
[32mINFO    [0m src.node.waku_node:waku_node.py:163 RLN credentials not set or credential store not available, starting without RLN
[35mDEBUG   [0m src.node.waku_node:waku_node.py:165 Using volumes []
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:49 docker run -i -t -p 46938:46938 -p 46939:46939 -p 46940:46940 -p 46941:46941 -p 46942:46942 wakuorg/nwaku:latest --listen-address=0.0.0.0 --rest=true --rest-admin=true --websocket-support=true --log-level=TRACE --rest-relay-cache-capacity=100 --websocket-port=46940 --rest-port=46938 --tcp-port=46939 --discv5-udp-port=46941 --rest-address=0.0.0.0 --nat=extip:172.18.203.244 --peer-exchange=true --discv5-discovery=true --cluster-id=3 --nodekey=ee8dabbb47ccac5573a3459f8ff6e7ac35c1bcf4d2868283ab7d58b2b48661fe --shard=0 --metrics-server=true --metrics-server-address=0.0.0.0 --metrics-server-port=46942 --metrics-logging=true --discv5-bootstrap-node=enr:-L24QBv2HcY4CFFxvgoZDWxwdW7-06Fg7-mb_HCn9rj-wPvST2UtJoSBaKKD6uStOn1_E9vsGWcb8EHT32PR1BbtL-oCgmlkgnY0gmlwhKwSvQmKbXVsdGlhZGRyc5YACASsEr0JBjA4AAoErBK9CQYwOd0DgnJzhQADAQAAiXNlY3AyNTZrMaEDGVbFSCUTFtsSKYOcWdvzIKRNq0t9hlEP75q9u_ArWH-DdGNwgjA4g3VkcIIwOoV3YWt1Mg0 --lightpushnode=/ip4/172.18.189.9/tcp/12344/p2p/16Uiu2HAmEMuHPouTm1aMEyDqseFEiEK9WMHoisqCqeYX82MwTe3g --lightpush=true --relay=true
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:55 docker network connect --ip 172.18.203.244 waku 08c543daddaa1520fa70f9f118248ca369c6c7a7b1a992b88df19db51ba37eeb
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:58 Container started with ID 08c543daddaa. Setting up logs at ./log/docker/lightpush_node2_2024-10-29_04-08-48__1fdfe0dc-43de-4e01-8b69-3c893b9a442e__wakuorg_nwaku:latest.log
[35mDEBUG   [0m src.node.waku_node:waku_node.py:177 Started container from image wakuorg/nwaku:latest. REST: 46938
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:46938/health" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"nodeHealth":"Ready","protocolsHealth":[]}'
[32mINFO    [0m src.node.waku_node:waku_node.py:277 Node protocols are initialized !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:46938/debug/v1/info" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"listenAddresses":["/ip4/172.18.203.244/tcp/46939/p2p/16Uiu2HAmVSRqG9EZFusjdtyja1a4GiQihu6UtsfCfUTphqZpZ6Ai","/ip4/172.18.203.244/tcp/46940/ws/p2p/16Uiu2HAmVSRqG9EZFusjdtyja1a4GiQihu6UtsfCfUTphqZpZ6Ai"],"enrUri":"enr:-L24QNOCaR9g2a9UyBw74xxZWUfAzNUNEBbRXdemkM1UkTC_UWWGSiIKhKqFf3uMOOaZyODYVaKHUcW_ml6-SpVSggECgmlkgnY0gmlwhKwSy_SKbXVsdGlhZGRyc5YACASsEsv0BrdbAAoErBLL9Aa3XN0DgnJzhQADAQAAiXNlY3AyNTZrMaED-V7wrHLbZVVPFZQiwLooGCqdSLiQy4_wjNd9pHEicdeDdGNwgrdbg3VkcIK3XYV3YWt1Mgk"}'
[32mINFO    [0m src.node.waku_node:waku_node.py:282 REST service is ready !!
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:46938/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.189.9/tcp/12344/p2p/16Uiu2HAmEMuHPouTm1aMEyDqseFEiEK9WMHoisqCqeYX82MwTe3g"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:46938/admin/v1/peers" -H "Content-Type: application/json" -d '["/ip4/172.18.180.221/tcp/28606/p2p/16Uiu2HAm2gpPkExoZnGzAWk3kh9VwZjDVwrwH4tqSizk24xZK3Eu"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:12343/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/0"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:46938/relay/v1/subscriptions" -H "Content-Type: application/json" -d '["/waku/2/rs/3/0"]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:28605/filter/v2/subscriptions" -H "Content-Type: application/json" -d '{"requestId": "1", "contentFilters": ["/myapp/1/latest/proto"], "pubsubTopic": "/waku/2/rs/3/0"}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'{"requestId":"1","statusDesc":"OK"}'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 1 seconds
[35mDEBUG   [0m src.steps.light_push:light_push.py:120 Lightpushing message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:13189/lightpush/v1/message" -H "Content-Type: application/json" -d '{"pubsubTopic": "/waku/2/rs/3/0", "message": {"payload": "TGlnaHQgcHVzaCB3b3JrcyEh", "contentTopic": "/myapp/1/latest/proto", "timestamp": 1730174946433399040}}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.1 seconds
[35mDEBUG   [0m src.steps.light_push:light_push.py:124 Checking that peer NODE_1:wakuorg/nwaku:latest can find the lightpushed message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:12343/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"TGlnaHQgcHVzaCB3b3JrcyEh","contentTopic":"/myapp/1/latest/proto","version":0,"timestamp":1730174946433399040,"ephemeral":false}]'
[35mDEBUG   [0m src.steps.light_push:light_push.py:124 Checking that peer NODE_2:wakuorg/nwaku:latest can find the lightpushed message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:46938/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"TGlnaHQgcHVzaCB3b3JrcyEh","contentTopic":"/myapp/1/latest/proto","version":0,"timestamp":1730174946433399040,"ephemeral":false}]'
[35mDEBUG   [0m src.steps.light_push:light_push.py:120 Lightpushing message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X POST "http://127.0.0.1:46938/lightpush/v1/message" -H "Content-Type: application/json" -d '{"pubsubTopic": "/waku/2/rs/3/0", "message": {"payload": "TGlnaHQgcHVzaCB3b3JrcyEh", "contentTopic": "/myapp/1/latest/proto", "timestamp": 1730174946558023936}}'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'OK'
[35mDEBUG   [0m src.libs.common:common.py:35 Sleeping for 0.1 seconds
[35mDEBUG   [0m src.steps.light_push:light_push.py:124 Checking that peer NODE_1:wakuorg/nwaku:latest can find the lightpushed message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:12343/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"TGlnaHQgcHVzaCB3b3JrcyEh","contentTopic":"/myapp/1/latest/proto","version":0,"timestamp":1730174946558023936,"ephemeral":false}]'
[35mDEBUG   [0m src.steps.light_push:light_push.py:124 Checking that peer NODE_2:wakuorg/nwaku:latest can find the lightpushed message
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:46938/relay/v1/messages/%2Fwaku%2F2%2Frs%2F3%2F0" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"TGlnaHQgcHVzaCB3b3JrcyEh","contentTopic":"/myapp/1/latest/proto","version":0,"timestamp":1730174946558023936,"ephemeral":false}]'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:37 curl -v -X GET "http://127.0.0.1:28605/filter/v2/messages/%2Fmyapp%2F1%2Flatest%2Fproto" -H "Content-Type: application/json" -d 'None'
[32mINFO    [0m src.node.api_clients.base_client:base_client.py:22 Response status code: 200. Response content: b'[{"payload":"TGlnaHQgcHVzaCB3b3JrcyEh","contentTopic":"/myapp/1/latest/proto","version":0,"timestamp":1730174946433399040,"ephemeral":false},{"payload":"TGlnaHQgcHVzaCB3b3JrcyEh","contentTopic":"/myapp/1/latest/proto","version":0,"timestamp":1730174946558023936,"ephemeral":false}]'
[35mDEBUG   [0m tests.conftest:conftest.py:59 Running fixture teardown: test_setup
[35mDEBUG   [0m tests.conftest:conftest.py:83 Running fixture teardown: close_open_nodes
[35mDEBUG   [0m src.node.waku_node:waku_node.py:221 Stopping container with id 3b76270051ef
[35mDEBUG   [0m src.node.waku_node:waku_node.py:228 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:221 Stopping container with id f0628d34ade2
[1m[31mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container 3b76270051ef. Exiting log stream.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:228 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:221 Stopping container with id b88318455ce1
[1m[31mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container f0628d34ade2. Exiting log stream.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:228 Container stopped.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:221 Stopping container with id 08c543daddaa
[1m[31mERROR   [0m src.node.docker_mananger:docker_mananger.py:89 Max retries reached for container b88318455ce1. Exiting log stream.
[35mDEBUG   [0m src.node.waku_node:waku_node.py:228 Container stopped.
[35mDEBUG   [0m tests.conftest:conftest.py:98 Running fixture teardown: check_waku_log_errors
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.
[35mDEBUG   [0m src.node.docker_mananger:docker_mananger.py:144 No errors found in the waku logs.